op-wheel/commands.go

```diff
@@ -271,6 +271,17 @@ var (
 			return ch.RunAndClose(cheat.SetBalance(addrFlagValue("address", ctx), bigFlagValue("balance", ctx)))
 		}),
 	}
+	CheatSetCodeCmd = cli.Command{
+		Name: "code",
+		Flags: []cli.Flag{
+			DataDirFlag,
+			addrFlag("address", "Address to change code of"),
+			cli.StringFlag{Name: "code", Usage: "New code of the account"},
+		},
+		Action: CheatAction(false, func(ctx *cli.Context, ch *cheat.Cheater) error {
+			return ch.RunAndClose(cheat.SetCode(addrFlagValue("address", ctx), ctx.String("code")))
```

Comment:
Similar to `addrFlag` helper, we should define a `hexutils.Bytes` generic-flag creation helper, to make bytes-flags easy and safe (errors are handled as part of flag-parsing then, and don't cause panics).